### ISSUE TITLE
[Feature request] Parameter for email report in environment

### ISSUE THREAD

**Issue by update-freak** [jlesage/docker-filebot#95]
### Idea

Hi, 
is it possible to send a report to a  specific email adress when the files are renamed/moved?
It's mentioned here: https://www.filebot.net/forums/viewtopic.php?t=215
I think this feature would be very helpful to check the correct behaviour / naming of FileBot.

**Comment by jlesage** [comment]
Yes, you can use the `AMC_CUSTOM_OPTIONS` environment variable to set any additional parameters.
See https://github.com/jlesage/docker-filebot#environment-variables

**Comment by update-freak** [comment]
ok, thanks. I tried with 
-e AMC_CUSTOM_OPTIONS=--def mailto="beispiel@gmx.de"  \
but this creates an error on Synology NAS Task schedule to create the docker container.

Is it then also necessary to define the following:
--def mail=host:port:from:username:password
or is the command mailto enough?

**Comment by jlesage** [comment]
First, since you are creating the container from shell, try to wrap the parameter around single quotes like this.  Also, I'm not sure the double quotes around the email are needed.
```
-e 'AMC_CUSTOM_OPTIONS=--def mailto=beispiel@gmx.de'
```

> Is it then also necessary to define the following:
> --def mail=host:port:from:username:password

I never used these options, but I would say yes.  FileBot needs an SMTP server to use to send emails.  You have to add it to `AMC_CUSTOM_OPTIONS`:

```
-e 'AMC_CUSTOM_OPTIONS=--def mailto=beispiel@gmx.de --def mail=host:port:from:username:password'
```

**Comment by update-freak** [comment]
Thank you. I got it working with

`docker run -d --name=FileBot \
-p 5802:5800 \
-e USER_ID=1026 \
-e GROUP_ID=100 \
-e LANG=de_DE.UTF-8 \
-e TZ=Europe/Berlin \
-e AMC_ACTION=move \
-e AMC_CONFLICT=skip \
-e AMC_LANG=German \
-e AMC_MOVIE_FORMAT="Filme/{n}/{n} ({y})" \
-e AMC_SERIES_FORMAT="Serien/{n}/Staffel {s}/{n} - {s00e00} - {t}" \
-e AMC_PROCESS_MUSIC=n \
-e AMC_CUSTOM_OPTIONS="--def mail=mail.gmx.net:587:beispiel@gmx.de:PASSWORD --def mailto=beispiel@gmx.de" \
-e AMC_INPUT_DIR=/storage/docker/jdownloader2/output/extracted \
-e AMC_OUTPUT_DIR=/storage/docker/jdownloader2/output/test \
-v /volume1/docker/filebot:/config:rw \
-v /volume1:/storage:rw \
jlesage/filebot`

Setup of the docker container was then ok.
But I got this error. but this should not be linked to an issue with this container.
[amc ] Sending Email report
[amc ] 530 Authentication required

So I will close this ticket then.
